src/providers/wfs/qgsbasenetworkrequest.cpp

```diff
@@ -731,6 +731,10 @@ void QgsBaseNetworkRequest::replyFinished()
                             .arg( exception.attribute( QStringLiteral( "exceptionCode" ), tr( "missing" ) ), exception.firstChildElement( QStringLiteral( "ExceptionText" ) ).text() );
         }
       }
+      else if ( !replyContent.isEmpty() )
+      {
+        mErrorMessage += tr( "\nserver response: %1" ).arg( replyContent );
```

Comment:
Not sure about it, but maybe we should start with the capital letter here?
```suggestion
        mErrorMessage += tr( "\nServer response: %1" ).arg( replyContent );
```